taxi/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Driver, Manufacturer
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
```

Comment:
Looks like your screenshots are outdated. -_-

Reply:
I've checked recently and nothing changed

Reply:
There should be a filter field in the car list. If there isn't, it means that there's something wrong.

Reply:
I understand, but I don't understand, why there is no this field
Thanks, I'll try to do smth with it

Reply:
Everything is ok)
I just added a second manufacturer)))))
<img width="1433" alt="Screenshot 2022-11-01 at 12 27 47" src="https://user-images.githubusercontent.com/111740601/199213560-357bc8cb-b57b-4c45-b78f-73219df71627.png">